catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/solr/FilteringSolrIndex.java

```diff
@@ -54,7 +54,7 @@
 
 public class FilteringSolrIndex {
 
-    private static IndexSchema indexSchema;
+    private static volatile IndexSchema indexSchema;
```

Comment:
Interesting. You should I get the following error when using the Intellij plugin or the Maven plugin:
> Incorrect lazy initialization of static field org.codice.ddf.ui.searchui.query.solr.FilteringSolrIndex.indexSchema in org.codice.ddf.ui.searchui.query.solr.FilteringSolrIndex.createSolrServer(String, ConfigurationFileProxy) [org.codice.ddf.ui.searchui.query.solr.FilteringSolrIndex] At FilteringSolrIndex.java:[lines 100-102]